dotCMS/src/main/java/com/dotmarketing/util/TaskLocatorUtil.java

```diff
@@ -282,6 +282,7 @@ public static void removeFixTask(Class<?> clazz){
         .add(Task05300UpdateIndexNameLength.class)
         .add(Task05305AddPushPublishFilterColumn.class)
 		.add(Task05350AddDotSaltClusterColumn.class)
+		.add(Task05370ChangeContainerPathToAbsolute.class)
```

Comment:
put at the end and naming according to the last task

Reply:
done https://github.com/dotCMS/core/pull/19122/commits/fbaded83a3eb9b46bb0c29d64e15db5ac60a772c#diff-a5f6335b9d67b8de8cbdc378cbe131a4R286